packages/typespec-vscode/src/utils.ts

```diff
@@ -230,21 +231,18 @@ export function spawnExecution(
   env?: NodeJS.ProcessEnv,
   on?: spawnExecutionEvents,
 ): Promise<ExecOutput> {
-  const shell = process.platform === "win32";
-  const cmd = shell && exe.includes(" ") ? `"${exe}"` : exe;
   let stdout = "";
```

Comment:
the change will impact all the code spawn execution. please ask vender to run a test pass to make sure it doesnt break anything.